antarest/study/storage/variantstudy/model/model.py

```diff
@@ -1,22 +1,49 @@
 import typing as t
+import uuid
 
+import typing_extensions as te
 from pydantic import BaseModel
 
 from antarest.core.model import JSON
 from antarest.study.model import StudyMetadataDTO
 
+LegacyDetailsDTO = t.Tuple[str, bool, str]
```

Comment:
Why did you not use a TypedDict here too ? Doing so allows you to make NewDetailsDTO inherit from this class so it could be nice.

Reply:
Actually, a `tuple` is not a `dict`.

Reply:
Yes you're right I've made that comment before reading all the code.